data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,31 @@ mission "Little Vienna in New Austria"
 			label backtoship
 			`	With the atmosphere steeped in nostalgia for a distant city on a distant world long past, you return to your ship.`
 
+mission "Easter Celebrations"
+	minor
+	source
+		planet "Harmony" "New Tortuga"
+	to offer
+		# Easter Sunday is on a Sunday.
+		"days since epoch" % 7 == 0
+		# Easter Sunday occurs between 1 to 7 days inclusive after the last paschal full moon.
+		# Days are counted from the end of the year to ensure consistency with leap years.
+		365 - "days until year end" > ( year % 19 * 19 + ( 15 - ( 13 + year / 100 * 8 ) / 25 + year / 100 - year / 400 ) % 30 ) % 30 + 80
+		365 - "days until year end" <= ( year % 19 * 19 + ( 15 - ( 13 + year / 100 * 8 ) / 25 + year / 100 - year / 400 ) % 30 ) % 30 + 80 + 7
+	on offer
+		conversation
+			`During a walk through the spaceport, you observe a small church tucked between a grain silo and a welding shop. Written on the door in large letters are the words "Stop by for a celebration of the Risen God!"`
+			choice
+				`	(Step in.)`
+				`	(Move on.)`
+					decline
+			`	Inside the building, a group is singing with deafening loudness. On the stage at the front of the room, a man strums a guitar; behind him a lady softly plays the piano. A large screen displays lyrics to the current song, which you note is "Christ Our Lord has Risen Again."`
```

Comment:
Hmm... would FL definitely know this song? Or would it be better to just say

```suggestion
			`	Inside the building, a group is singing with deafening loudness. On the stage at the front of the room, a man strums a guitar; behind him a lady softly plays the piano. A large screen displays lyrics to the current song, "Christ Our Lord has Risen Again."`
```

and suggest that the title is on the screen?